app/views/task_contributions/new.html.slim

```diff
@@ -0,0 +1,5 @@
+.header
+  .std-heading
+    i.fa-regular.fa-file-lines
+    =< + t('.new_contribution')
```

Comment:
With #1359, Zuhra added some further nesting and other classes here, which you might want to copy, too:

```suggestion
.header
   .row
     .col-md-12.pb-2.border-bottom
       .std-heading
         i.fa-regular.fa-file-lines
         =< + t('.new_contribution')
```

Reply:
Since the headline is not shown correctly (see some comment above), which is caused by these lines, I am unresolving this thread, too.

Reply:
Actually, I saw that I made a specific (already resolved 🤓) comment about that issue, which is more descriptive. Hence, I am resolving the current discussion and rather unresolving [this one](https://github.com/openHPI/codeharbor/pull/1103/files#diff-e82418b173e8baf2d2e214e67edec3903491ca7c16f9831d04608b893ff96d8f).